server/test/services/yeelight/lib/device/setValue.test.js

```diff
@@ -0,0 +1,39 @@
+const { assert } = require('chai');
+const proxyquire = require('proxyquire').noCallThru();
+const GladysDevice = require('../../Gladys-color.json');
+const YeelightApi = require('../../yeelight.mock.test');
+
+const YeelightService = proxyquire('../../../../../services/yeelight/index', {
+  'yeelight-awesome': YeelightApi,
+});
+
+describe('YeelightHandler setValue', () => {
+  const yeelightService = YeelightService({}, 'a810b8db-6d04-4697-bed3-c4b72c996279');
+
+  it('should set binary value', async () => {
+    await yeelightService.device.setValue(GladysDevice, { category: 'light', type: 'binary' }, 1);
+  });
+  it('should set brightness value', async () => {
+    await yeelightService.device.setValue(GladysDevice, { category: 'light', type: 'brightness' }, 90);
```

Comment:
Maybe you can test that nothing happen (with not called)?

Reply:
Idem previous comment.

Reply:
Done